models/pod.go

```diff
@@ -127,15 +127,19 @@ func lookupImage(containerName string, containers []core_v1.Container) string {
 	return ""
 }
 
-func (pods Pods) HasIstioSideCar() bool {
-	for _, pod := range pods {
-		if pod.HasIstioSideCar() {
-			return true
+// HasIstioSidecar returns true if there are no pods or all pods have a sidecar
```

Comment:
Ok, so the logic is reversed and it took me a while to figure out, at first glance it looked incorrect to me because now we return false if at least one pod has no sidecar. Before we were returning true if there was at least one sidecar.

This is indeed what we want, as this is used to report any missing sidecar in the UI. Perhaps in some places the variables are not named very accurately and this is why it can be confusing. Like "HasSidecar" is actually "HasNoMissingSidecar", that's a slight difference :-)